src/VisualStudio/CSharp/Impl/Options/AdvancedOptionPageControl.xaml.cs

```diff
@@ -81,39 +83,43 @@ public AdvancedOptionPageControl(OptionStore optionStore, IComponentModel compon
                 return false;
             });
 
-            BindToOption(Split_string_literals_on_enter, SplitStringLiteralOptions.Enabled, LanguageNames.CSharp);
```

Comment:
super frustrated with having to maintain this code and figure out where things should go.  now the order of hte code matches the order of hte groups and items in the xaml.